src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -1181,9 +1180,13 @@ public TypeElement getSuperClass(TypeElement te) {
         return asTypeElement(superclass);
     }
 
+    private boolean checkType(TypeElement te) {
```

Comment:
The name isn't great, and seems to be more general than the implementation might suggest.

Reply:
I agree; could you try to come up with a better one?